modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Allow removing table classes via the table props dialog to keep consistent with how the cell and row class selecting works. #TINY-6653
```

Comment:
Can this somehow describe that the actual change is here. Like "It was not possible to remove the class attribute from table elements by selecting empty string items configured by the `table_class_list` option." or something. Since just referring to what the cell and row dialog then you need to know how those work.